ui-asset-management/api/svc-asset-count/extension/clients/asset_count_client.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024 Paladin Cloud, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package clients
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"svc-asset-count-layer/models"
+)
+
+type AssetCountClient struct {
+	elasticSearchClient *ElasticSearchClient
+	rdsClient           *RdsClient
+}
+
+func NewAssetCountClient(ctx context.Context, config *Configuration) *AssetCountClient {
+	dynamodbClient, _ := NewDynamoDBClient(ctx, config.UseAssumeRole, config.AssumeRoleArn, config.Region, config.TenantConfigOutputTable, config.TenantTablePartitionKey)
+	secretsClient, _ := NewSecretsClient(ctx, config.UseAssumeRole, config.AssumeRoleArn, config.Region)
+
+	return &AssetCountClient{
+		elasticSearchClient: NewElasticSearchClient(dynamodbClient),
+		rdsClient:           NewRdsClient(secretsClient, config.SecretIdPrefix),
+	}
+}
+
+const (
+	success     = "success"
+	awsProvider = "aws"
```

Comment:
I think we can use the column type='Cloud Provider' because the plugins table doesnt have an isCloud column

![image](https://github.com/user-attachments/assets/6a68733e-a651-4a14-a890-b932a0b28aed)

Reply:
done